src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -82,19 +82,42 @@ def compress(
         """
         compressed_dict = {}
         weight_suffix = ".weight"
+        input_zp_suffix = ".input_zero_point"
+        weight_zp_suffix = ".weight_zero_point"
         _LOGGER.debug(
             f"Compressing model with {len(model_state)} parameterized layers..."
         )
 
         for name, value in tqdm(model_state.items(), desc="Quantized Compression"):
+            # check if the parameter we're compressing is the weight zp
+            # or the input zp
+            weight_zp = name.endswith(weight_zp_suffix)
```

Comment:
might help with readability
```suggestion
            is_weight_zp = name.endswith(weight_zp_suffix)
```